lib/modules/datasource/github-releases/cache/cache-base.ts

```diff
@@ -365,4 +377,25 @@ export abstract class AbstractGithubDatasourceCache<
 
     return result;
   }
+
+  public isChangelogReleaseFresh(
+    changelogRelease: ChangeLogRelease | undefined,
+    now: DateTime,
+    updateDuration: DurationLike,
+    cacheItems: Record<string, StoredItem>
+  ): boolean {
+    if (changelogRelease) {
+      const changelogReleaseTime = DateTime.fromISO(
+        changelogRelease.date.toString()
```

Comment:
Not all releases have a timestamp from datasource. Does that mean changelogRelease will be null to start with?

Reply:
I think there is nothing we can do about when `releaseTimestamp` is empty

Reply:
I understand, just wanted to check that this code doesn't cause error?

Reply:
Thanks, it really was a bug, fixed for now